database/migrations/2016_04_01_134534_create_users_tickets_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateUsersTicketsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_ticket', function (Blueprint $table) {
```

Comment:
By convention and for laravel magic to work, table name should be **'ticket_user**'.
----
check this example found in : https://laravel.com/docs/5.1/eloquent-relationships

To define [.., the many to many relationship between a user and a role], three database tables are needed: users, roles, and role_user. The **role_user** table is derived from the **alphabetical order of the related model names**, and contains the user_id and role_id columns.